.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
-  - '0.10'
-  - '0.11'
-  - '0.12'
-  - 'iojs-1'
-  - 'iojs-2'
+  - '4'
+  - '5'
+  - '6'
```

Comment:
On last change, can you remove v4 and v5, those are EOL by node so I'd like to remove them from this major release as well.